Helper/SeedData.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Helpers.Http;
+using System.Net;
+
+namespace Helpers
+{
+    public static class SeedData
+    {
+        private const string STORAGE_ADAPTER_ADDRESS = "http://127.0.0.1:9022/v1";
+        private const int TIMEOUT = 10000;
+        private static HttpClient httpClient = new HttpClient();
+
+        public static void WaitForSeedComplete()
```

Comment:
what happens if seed has failed completely? This test will run forever right? I would add a max retry or something